Jenkinsfile

```diff
@@ -477,7 +477,7 @@ pipeline {
         stage('build-gui-clang-qt5') {
           agent {
             docker {
-              image 'docker.openmodelica.org/build-deps:v1.22.2'
+              image 'anheuermann/openmodelica-build-deps:v1.22.1-qtwebengine'
```

Comment:
It is changed here. 

I am not sure what it is but there is something not quite right in this PR on Jenkins. As I mentioned it is also not picking up your changes to the `Jenkinsfile` regarding the CMake build. 

We can try rebasing it on master if you have not done so yet. Let me know if you want to try that.

Reply:
done